signing_service/signing_service/utils.py

```diff
@@ -96,21 +98,26 @@ def __call__(  # type: ignore
         self,
         parser: ArgumentParser,
         namespace: Namespace,
-        values: str,
+        value: str,
         option_string: Optional[str]=None
     ) -> None:
-        if values is not None and self.use_file:
-            with open(values) as file:
+        assert value is not None
+        if self.use_file:
+            with open(value) as file:
                 self.const = file.read()
-        elif values is not None and self.read_command_line:
-            self.const = values
+        elif self.read_command_line:
+            self.const = value
         elif self.env_variable_name is not None:
             self.const = os.environ.get(self.env_variable_name)
         else:
             assert False
```

Comment:
this is not your change, but consider instead of `assert False` raise some exception, may be custom. And even better last `elif` change to `else` and add `assert self.env_variable_name is not None`

```python
 else:
    assert self.env_variable_name is not None:
    self.const = os.environ.get(self.env_variable_name)
```

Reply:
Personally I would change it to what You propose, but as said, it's not my code originally, so some second opinion is needed. @rwrzesien ? @cameel ?

Reply:
I personally don't see much difference in functionality and logic between those two so up to you.

I also don't really understand what should be done here. First the suggestion is about raising exception, then using assert, so I don't know what is the final solution you want to apply here.

Reply:
Okey, I've change it.